pkg/controller/grafana/grafana_controller.go

```diff
@@ -240,6 +252,12 @@ func (r *ReconcileGrafana) configureGrafanaDashboard(route *openshiftv1.Route) e
 	return nil
 }
 
+func GetCredentialedHostPathUrl(secret *corev1.Secret, host string, path string) string {
+	user := secret.Data["GF_SECURITY_ADMIN_USER"]
+	pass := secret.Data["GF_SECURITY_ADMIN_PASSWORD"]
+	return fmt.Sprintf("https://%s:%s@%s%s", strings.TrimSpace(string(user)), strings.TrimSpace(string(pass)), host, path)
```

Comment:
Quick question. Does the secret `Data` slice have Base64 encoded values in it? If so, would we need to decode them before inserting them into the URL?

Reply:
Yea, I was a bit confused about that after reading the docs, but when I tested it manually I found that when reading/writing from the `Secrets` object, the data seemed to be automatically encoded/decoded. However when you view the contents of the secret with ex. `oc get -o json secret foo`, it's still encoded. I'll double check all that though.

Reply:
Just checked by adding in a log statement when the grafana_controller reads the credentials here and they are indeed already decoded at this point, but when you do an `oc get secret` it's still encoded.

Reply:
Thanks for checking. The documentation was a bit unclear about that, and I couldn't find where the decoding happens. Seems good to go from me.